### PR TITLE
Sympy abs is better for handling complex number #9474

### PR DESCRIPTION
**Pull request by Shekharrajak**
Fixes https://github.com/sympy/sympy/issues/9474
module should be default.
I think math module  can handle only complex number that contains I(Iota).

math.fabs() converts its argument to float if it can (if it can't, it throws an exception). It then takes the absolute value, and returns the result as a float.

```
>>> type(abs(-2))
<type 'int'>
>>> type(abs(-2j))
<type 'float'>
>>> import math
>>> type(math.fabs(-2j))
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: can't convert complex to float
```

### CODE REVIEW

**Comment by Shekharrajak** [comment]
It seems some of the math functions are not in mpmath.I will try to change module for particular tests.

**Comment by Shekharrajak** [comment]
Another way is use cmath module for complex number instead of math module. [Refer](http://hplgit.github.io/primer.html/doc/pub/formulas/._formulas-bootstrap006.html)

**Comment by moorepants** [comment]
Please change your PR title to something that is informative.

**Comment by Shekharrajak** [comment]
Done @moorepants

File: sympy/utilities/tests/test_lambdify.py [hunk 1]
```diff
@@ -581,3 +581,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+
+def test_issue_9474():
+    k = symbols('k', complex=True)
+    f = lambdify([k], abs(k**2))
+    assert f(10.01+0.1j) == 100.2101
```
Comment (smichr) [inline sympy/utilities/tests/test_lambdify.py]: This doesn't raise an error for me as in the issue.
What is asserted in this PR fails (but the str version does not).

```
>>> k = symbols('k', complex=True)
>>> f = lambdify([k], abs(k**2))
>>> f(10.01+0.1j)
100.21009999999998
>>> _==100.2101
False
>>> str(f(10.01+0.1j))
'100.2101'
```

**Comment by thisch** [comment]
This PR can be closed since #9474 has already been fixed in master

**Pull request closed without merging**